spacecmd/src/spacecmd/configchannel.py

```diff
@@ -809,12 +814,11 @@ def do_configchannel_addfile(self, args, update_path=''):
                 if options.channel in config_channels:
                     failures = 0
                     break
-                else:
-                    print('')
-                    logging.warning(_N('%s is not a valid channel') %
-                                    options.channel)
-                    print('')
-                    failures += 1
+                print('')
+                logging.warning(_N('%s is not a valid channel') %
+                                options.channel)
```

Comment:
Isn't this triggering a "logging not lazy" warning?

Reply:
Not that I noticed (which does not mean anything). I have to admit that I am not confident on what I am doing here. So I just took your comments as is.